apps/marketplace/components/BuyerATM/BuyerATMAdditionalInformationStage.js

```diff
@@ -66,7 +92,8 @@ class BuyerATMAdditionalInformationStage extends Component {
           defaultValue={this.props[this.props.model].contactNumber}
           maxLength={100}
           validators={{
-            required
+            required,
+            validPhoneNumber
```

Comment:
```suggestion
            requiredContactNumber,
            contactNumberFormat
```

Reply:
When I did this, have a negative effect on the contact box and it would find a valid phone number 'wrong' i.e it would become red but it was fine to submitted it. (See image below)
image

![image](https://user-images.githubusercontent.com/31645577/79168180-957ed700-7e2c-11ea-9060-dff106fb1b87.png)